assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -615,3 +615,11 @@ che.workspace.devfile.default_editor.plugins=eclipse/che-machine-exec-plugin/nig
 # which will be mount into workspace containers as a files or env variables.
 # Only secrets that match ALL given labels will be selected.
 che.workspace.provision.secret.labels=app.kubernetes.io/part-of=che.eclipse.org,app.kubernetes.io/component=workspace-secret
+
+
+# Plugin is added in case async storage feature will be enabled in workspace config
+# and supported by environment
+che.workspace.devfile.async.storage.plugin=eclipse/che-async-pv-plugin/nightly
+
+# Docker image for the Che async storage
+che.infra.kubernetes.async.storage.image=vparfonov/che-workspace-data-sync-storage:latest
```

Comment:
let's move it to https://quay.io/organization/eclipse

Reply:
done